lib/cmd/fh3/appdata/import/start.js

```diff
@@ -0,0 +1,87 @@
+"use strict";
+/* globals i18n */
+
+var async = require('async');
+var path = require('path');
+var mime = require('mime');
+
+var fs = require('fs')
+  , fhreq = require('../../../../utils/request.js')
+  , readline = require('readline')
+  , common = require('../../../../common');
+
+function retrieveUploadMetadata(envId, appId, filePath, cb) {
+  var url = 'api/v2/appdata/' + envId + '/' + appId + '/data/import';
+
+  fs.stat(filePath, function (err, stat) {
```

Comment:
it might not be a bad idea to check the file before upload?

e.g. it should be a tar file, it should contain a collection of .gz files?